subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -53,9 +90,36 @@ It's helpful to know the following terminology before you dive into the details
 
 Projects are the things that Gradle builds.
 Projects contain a build script, which is a file located in the project's root directory usually named `build.gradle` or `build.gradle.kts`.
+
 Builds scripts define tasks, dependencies, plugins, and other configuration for that project.
 A single build can contain one or more projects and each project can contain their own subprojects.
 
+=== Builds
+
+A build is an execution of a collection of tasks in a Gradle project.
```

Comment:
This contradicts the previous sentence:

> A single build can contain one or more projects and each project can contain their own subprojects.

What about the following?
```suggestion
A build invocation is an execution of a collection of tasks in the projects contained by a build.
```

Probably not the best sentence but at least the terminology is clearer to me. WDYT?

Reply:
I am thinking:
```suggestion
A build invocation is the execution of a collection of tasks in the project(s).
```
Thoughts?

Reply:
Sounds good! :+1:

Reply:
Updated in the latest commit.